### PR TITLE
Add actions guide

### PR DESCRIPTION
**Pull request by ekoleda-codaio**

### CODE REVIEW

File: docs/guides/blocks/actions.md [hunk 1]
```diff
@@ -2,6 +2,85 @@
 title: Actions
 ---
 
-# Add custom actions
+# Add custom actions for buttons and automations
 
-TODO
+In Coda users can take actions directly within their documents, using native [buttons][help_buttons] and [automations][help_automations]. Packs can provide custom actions which users can use when building out their doc. These actions often use the [fetcher][fetcher] to send data to an external API, but they can also be used for other one-time calculations.
```
Comment (jonathan-codaio) [inline docs/guides/blocks/actions.md:7]: `fetcher` link looks broken

File: docs/guides/blocks/actions.md [hunk 2]
```diff
@@ -2,6 +2,85 @@
...
+
+
+## Parameters
+
+Parameters in actions work the same as they do in other formulas, but may appear differently depending on the context. The button and automation configuration dialogs by default will present the parameters as input boxes, although the user has the option to switch the formula editor as well. See the [Parameters guide][parameters] for more information and examples.
+
+
+## Results
+
+Actions can return a value but aren't required to. When the action is used by a button in a table column, the result can be stored in another column of the same table. For this reason it's a best practice to return some value, so that other formulas or conditional formatting can check that column and know when the action is complete. For example, if your action is creating an item in an external API you could return the URL or ID of the new item.
```
Comment (jonathan-codaio) [inline docs/guides/blocks/actions.md:49]: Per Slack convo, would suggest also adding a line that if there isn't a meaningful unique value to return, consider just returning "OK" as an indicator that the action was completed successfully.

File: documentation/examples/action/dice.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    coda.makeParameter({
+      type: coda.ParameterType.Number,
+      name: "quantity",
+      description: "How many dice to roll.",
+      defaultValue: 1,
+    }),
+    coda.makeParameter({
+      type: coda.ParameterType.Number,
+      name: "sides",
+      description: "How many sides to the dice have.",
```
Comment (jonathan-codaio) [inline documentation/examples/action/dice.ts:19]: Nit, extra `to` I think.

**Review by jonathan-codaio — APPROVED** [review]
Nice

File: docs/guides/blocks/actions.md (hunk 1 above)
Comment (ekoleda-codaio) [reply docs/guides/blocks/actions.md:7]: Good catch, fixed.

File: docs/guides/blocks/actions.md (hunk 2 above)
Comment (ekoleda-codaio) [reply docs/guides/blocks/actions.md]: Done. PTAL.

File: documentation/examples/action/dice.ts (hunk 3 above)
Comment (ekoleda-codaio) [reply documentation/examples/action/dice.ts]: Fixed.

**Pull request merged**